src/Bot/Command/AbstractCommand.php

```diff
@@ -8,6 +8,7 @@
 namespace Kaecyra\ChatBot\Bot\Command;
 use Kaecyra\AppCommon\Event\EventAwareInterface;
 use Kaecyra\AppCommon\Event\EventAwareTrait;
+use Kaecyra\ChatBot\Client\Slack\Strategy\AbstractStrategy;
```

Comment:
I don't like this. Makes Bot depend on Slack.

Reply:
Hmm, I agree, the thing is, it was already implicitly depending on Bot in a way.

https://github.com/kaecyra/chatbot/blob/7d11cd5e60c38fe21b2024dcbb7d5dd166cac2be/src/Client/Slack/SlackRtmClient.php#L644-L675

That `->strategy` property is used dynamically. The question I'm asking myself right now is should strategies really be under the slack namespace? I mean they aren't really client related?

Reply:
Will move the abstract strategy under Bot.